src/content/docs/codestream/codestream-ui-overview/my-organization.mdx

```diff
@@ -1,47 +1,51 @@
 ---
-title: Manage My Organization on CodeStream
+title: Manage your organization on CodeStream
 metaDescription: "The My Organization menu provides tools for managing the members of your team, as well as the emails used by your teammates when committing code."
 ---
 
-The My Organization menu allows you to see who is in your CodeStream
-organization, invite new members, and create blame maps.
+New Relic CodeStream provides a number of tools for managing your organization.
+
+## The organization menu [#my-organization]
 
 ![My Organization Menu](./images/MyOrgMenu.png)
 
-## My Organization
+In the header, click the **My Organization** menu button to see who's in your New Relic CodeStream
+organization, invite new members, and create blame maps. 
 
-Select "My Organization" to see a list of all members of your CodeStream
-organization. Admins will be identified in the list, and if you are an admin,
-you'll have a dropdown to assign or remove admin priviledges to any member.
+Admins are identified in the teammates list. If you're an admin,
+there's a dropdown to assign or remove admin privileges to any member.
 
 ![Organization Members](./images/MyOrgMembers.png)
 
-## Invite Teammates
+## Invite your teammates [#invite]
 
-Select "Invite Teammates" to invite new members to your organization. 
+Click **Invite Teammates** to invite new members to your organization. 
 
 ![Reinvite](./images/MyOrgInvite.png)
 
-The "Outstanding Invitations" section lists all open invitations, and at the
-right side of each row are links to remove the invitation or to reinvite.
-Clicking "reinvite" will have CodeStream send another invitation via email, but
-you can also hover over the reinvite link for the option to generate an email
+The outstanding invitations section lists all open invitations. The
+right side of each row has links to remove the invitation or to resend the invite.
+
+Click **reinvite** to send another invitation via email. You
+can also hover over the reinvite link for the option to generate an email
 yourself.
 
-The "Suggested Teammates" section, which is only available for admins, is a list
+The suggested teammates section, only available for admins, is a list
 of possible teammates derived from the commit history of your open repositories.
 At the right side of each row are links to remove the suggestion from the list
 or to invite the person.
 
-## Blame Map [#blame]
+## Blame map [#blame]
 
-Select "Blame Map" to add email addresses that you use for committing code that
+Click **Blame Map** to add email addresses that you use for committing code that
 may be different from the email address you used to sign up for CodeStream. For
 example, your CodeStream email address might be dave@acme.com, but you might
-also commit code as dave@webmail.com. Click "Add mapping", enter your git email
+also commit code as dave@webmail.com. 
```

Comment:
I've always put e-mails in code segments. Maybe that's bad practice, but thought I'd bring that up!

Reply:
No, it's a good call. Not sure we even need an example of what an email address looks like :D